Makefile

```
@@ -0,0 +1,31 @@
+COMPOSE_FILE := local-env.yml
+LOGS_DIR := logs
+SERVICES := proof-server indexer node
+
+.PHONY: env-up env-down env-logs env-logs-clean env-status
+
+## Start local environment and stream logs to logs/
+env-up: env-down
```

checkmake (0.3.2)

[warning] Target body for "env-up" exceeds allowed length of 5 lines (6).

(maxbodylength)